api/app/files/api.py

```diff
@@ -45,9 +46,33 @@ def get(self):
             return send_file(temp.name, as_attachment=True, attachment_filename=args['filename'], mimetype='application/pdf')
 
 
+
+class FileAPIImproved(FileUploadMixin, restful.Resource):
+    def get(self, file_id):

```

Comment:
Remove the `file_id` argument. We use the request parser to extract the arguments from the HTTP request query.